models/erfc_neuron.h

```diff
@@ -0,0 +1,142 @@
+/*
+ *  erfc_neuron.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef ERFC_NEURON_H
+#define ERFC_NEURON_H
+
+// Includes from models:
+#include "binary_neuron.h"
+#include "binary_neuron_impl.h"
+
+namespace nest
+{
+/* BeginDocumentation
+   Name: erfc_neuron - Binary stochastic neuron with complementary error
+   function as activation function.
+
+   Description:
+   The erfc_neuron is an implementation of a binary neuron that
+   is irregularly updated as Poisson time points. At each update
+   point the total synaptic input h into the neuron is summed up,
+   passed through a gain function g whose output is interpreted as
+   the probability of the neuron to be in the active (1) state.
+
+   The gain function g used here is
+   g(h) = 0.5 * erfc (( h - theta_ ) / ( sqrt( 2. ) * sigma)).
+   This corresponds to a McCulloch-Pitts neuron receiving additional
+   Gaussian noise with mean 0 and standard deviation sigma
+   The time constant tau_m is defined as the mean
+   inter-update-interval that is drawn from an exponential
```

Comment:
Is the mean interval drawn from the distribution? Aren't the intervals drawn from a distribution with this mean?